easybuild/easyconfigs/i/impi/impi-2021.10.0-intel-compilers-2023.2.1.eb

```
@@ -0,0 +1,16 @@
+name = 'impi'
+version = '2021.10.0'
+
+homepage = 'https://software.intel.com/content/www/us/en/develop/tools/mpi-library.html'
+description = "Intel MPI Library, compatible with MPICH ABI"
+
+toolchain = {'name': 'intel-compilers', 'version': '2023.2.1'}
+
+# see https://software.intel.com/content/www/us/en/develop/articles/oneapi-standalone-components.html
+source_urls = ['https://registrationcenter-download.intel.com/akdlm/IRC_NAS/4f5871da-0533-4f62-b563-905edfb2e9b7/']
+sources = ['l_mpi_oneapi_p_%(version)s.49374_offline.sh']
+checksums = ['ab2e97d87b139201a2e7dab9a61ac6e8927b7783b459358c4ad69a1b1c064f40']
+
+dependencies = [('UCX', '1.15.0-rc4')]
```

Hmm, not sure it's wise to use a release candidate, we should stick to 1.14.1 instead, no?

---

Should change to 1.14.1 once #18877 is merged

```suggestion
dependencies = [('UCX', '1.14.1')]